### DISCUSSION TITLE
3.3v Multiplexer Options for Pico (Voltage Issues?)

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#278] [Q&A]
Hi!

I want to add a multiplexer to my project, which is based on a Raspberry Pi Pico.

I saw an earlier discussion here (as well as the Opendeck Custom Boards wiki) of how many of the multiplexers are built for 5v and not 3.5v, which is what the Pico provides.

I looked up the Pico and multiplexers on Adafruit and they have some listed that seem to be the standard 8 and 16 channel multiplexers.

This one on Amazon is identical, and it says it has an operating range of 2-6v, but it also says _“On” resistance: 70 Ohms @ 4.5V_

Does this mean that this mux is not going to work or will give me problems?

https://www.amazon.com/Ximimark-Digital-Multiplexer-Breakout-CD74HC4067/dp/B07K7JF3HX/ref=sr_1_3?crid=2D8JOH83XO97C&dib=eyJ2IjoiMSJ9.ej8Q16NkivCPBLOsWpDGxrf5nZSHQSCvpihc-tiBnWizc1StIqZs6gLg_K781LgSuZRTtlzzYGOL80ocxfn-Rybh6Zy1eHdMwQknzjiIGaGDDEKqVlN08DjpDfbZ7CUtvmz__DihAH3TlIvGkd_hZ-UwKjRzsyDM2_0qkZ-vYF0uS1VUvQKepf9dTh9WuXb4tyE9ygiwH78wvo72iu22Y66s4HjR0tVK2ynD66ub_9k.cAq2lO9O9l2QzR7yDIFyqMHaFYXKedNbunT3FI-TSvA&dib_tag=se&keywords=4067+multiplexer&qid=1709479469&sprefix=4067+multiplexer%2Caps%2C754&sr=8-3

Thanks!

![51YYMc46c8L _AC_](https://github.com/shanteacontrols/OpenDeck/assets/25620049/dddb34bb-c019-4ec6-a808-829b5d8bc478)

**Comment by paradajz** [comment]
As long as you use 74HCT125, you can use any multiplexer you want.

**Comment by NRKorsakov** [comment]
Thanks very much!
